### PR TITLE
[ignore] add hint to issue template context section

### PR DESCRIPTION
**Pull request by line-o**
add a hint that the context can be conveniently gathered using `xst info`
see https://github.com/eXist-db/xst/pull/49

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ If the above isn't working, please tell us the exact steps you took when you enc
 **Screenshots**
 If applicable, add screenshots to help explain your problem.
 
-**Context (please always complete the following information):**
+**Context (please always complete the following information, you can past the output of `xst info` here):**
```
Comment (adamretter) [inline .github/ISSUE_TEMPLATE/bug_report.md:83]: `past` -> `paste`.

Also could we improve the wording a little... perhaps?

```
information; one option is to use the [xst](URL GOES HERE) tool, and copy and paste the output produced by running `xst info` here.

**Review by dizzzz — CHANGES_REQUESTED** [review]
see comments

**Comment by line-o** [comment]
@dizzzz yes... I gave it a thumbs up already no need to get loud.

**Comment by line-o** [comment]
I will put this into draft until the actual feature in xst is available.

**Review by dizzzz — APPROVED** [review]

**Comment by line-o** [comment]
see `xst info` output in for a preview #4694

**Review by reinhapa — APPROVED** [review]

**Comment by line-o** [comment]
This PR is now open for 2 weeks and has two approvals.

**Comment by reinhapa** [comment]
@line-o seems that you need to update the current `develop` as here your #4696 seems to missing...

**Comment by line-o** [comment]
For this change as it only affects the GitHub issue template it makes no difference. This will solve itself by merging it into develop.

**Comment by line-o** [comment]
I do not have that branch checked out. I edited the issue template directly on github.

**Pull request merged**